menu_silver.rb

```diff
@@ -0,0 +1,21 @@
+adjectives = ['hot', 'cold', 'creamy', 'soft', 'hard', 'crunchy', 'warm', 'spicy', 'gelatinous',
```

Comment:
When you're defining a bunch of strings in an array, it's hard to keep formatting, spacing, quote styling the same, and not leaving dangling entries.

One way around it is to use `%w` or `%W`.

So for example, you could rewrite adjectives as:
```
adjectives = %w[ hot cold creamy soft hard crunchy warm 
                 spicy gelatinous chewy crispy juicy]
 => ["hot", "cold", "creamy", "soft", "hard", "crunchy", "warm", "spicy", "gelatinous", "chewy", "crispy", "juicy"]
```
And voila, it'll separate all the words at the spaces and do all the quoting for you.

But what if there are spaces in your word, like in foods? You can use `%W` with escapes (`\s` is space)
```
foods = %W[ chicken mashed\spotatoes  hamburger]
 => ["chicken", "mashed potatoes", "hamburger"] 
```